src/modules/Transcript/EventDetails/index.tsx

```diff
@@ -92,6 +107,24 @@ export function EventDetailsUI(props: EventDetailsUIProps): ReactElement {
                             </span>
                         </div>
                     )}
+                    {userQuery.state.data?.currentUser?.apiKey && (
+                        <div className="flex my-3 px-3.5">
+                            <span className="font-semibold flex-shrink-0 block w-28 mr-1">Transcript </span>
+                            <span className="block truncate">
+                                <a
+                                    className="text-blue-600 hover:text-blue-700 active:text-blue-800 hover:underline"
+                                    href={
+                                        `https://audio` +
+                                        (process.env.NODE_ENV !== 'production' ? `-dev` : '') +
```

Comment:
Where would this be defined for the mobile prod build? Is there an env file or config somewhere?

Reply:
@mmichael0413 the mobile build would be different?